.github/workflows/pull-request-comments.yml

```diff
@@ -128,6 +128,11 @@ jobs:
             const fs = require( 'fs' );
             const issue_number = Number( fs.readFileSync( './NR' ) );
 
+            // Don't comment for Dependabot.
+            if ( context.actor == 'dependabot' ) {
+              return;
+            }
+
```

Comment:
Can this instead be achieved with an `if` directive? For example:

```
if: ${{ github.actor != 'dependabot' }}
```

Reply:
Ah, yes. I think you're right. Updated to use that approach instead.